__tests__/app.test.js

```diff
@@ -94,7 +94,15 @@ describe("/api/articles", () => {
             expect(response.body.articles).toHaveLength(1);
         })
     })
-    test("GET: 404 responds with an error when no topics are found for the given query", () => {
+    test("GET: 404 responds with an error when no articles are found for the given topic query", () => {
+        return request(app)
+        .get("/api/articles?topic=paper")
+        .expect(404)
+        .then((response) => {
```

Comment:
This actually isn't a 404. the topic of paper is valid, there are just no associated articles. This should be status 200: responds with an empty array of articles